integration/fabric/iou/views/finality.go

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
Maybe we could reuse the view that we defined before?

Reply:
done